front_end/panels/profiler/HeapProfileView.ts

```diff
@@ -380,12 +381,22 @@ export class SamplingHeapProfileType extends SamplingHeapProfileTypeBase {
   }
 
   override startSampling(): void {
+    Host.rnPerfMetrics.heapSamplingStarted();
     const heapProfilerModel = this.obtainRecordingProfile();
     if (!heapProfilerModel) {
+      Host.rnPerfMetrics.heapSamplingFinished(
+          false,  // success
```

Comment:
nit: With IDE language tooling, we shouldn't need these param name comments.

Reply:
I'd prefer to keep it. WebStorm displays this nicely, but VS Code doesn't (vanilla without extensions); most importantly, browsing on GitHub is much better this way

Reply:
I get your point, but we can have developers configure this in VS Code if they need. No precedence for this in the react-native or fbsource codebases.

If we must keep these, please use exactly one space before ` //`.

Reply:
I've seen this pattern in our codebases. I personally always add them for boolean and numberic _literals_, becuase this is not helpful when readable on GitHub:

```
samplingFinished(true)
```

because boolean itself doesn't tell us much (as `true` could mean "failed" or "succeeded" as we've seen above).

While modern IDEs are sophisticated, we still do things that aren't strictly necessary, e.g. prefixing constants with `k` and/or using uppercase vs camelcase, even when IDEs colour-code them.

That said, there are a few alternatives that I consider in different cases, e.g.

* Use a named variable, even if it's a literal:
```
const success = true;
samplingFinished(success);
```
* Use separate functions for the success and fail case:
```
failed ? samplingFailed() : samplingFinished();
```
* Use an enum:
```
samplingFinished(failed ? Result.FAILED : Result.OK)
```
* Use an object:
```
samplingFinished({success: failed})
// or union
samplingFinished(error ? {success: false, errorMessage: error.message} : {success: true})
```

I think this can go into the wider discussion for our custom Prettier/code style guide. Existing formatter and code base use double spaces at the minimum. Let's revisit this and keep beefing up our CI (unit test is coming via #105!)